### PR TITLE
Refactor to vue3

### PR DESCRIPTION
**Pull request by pierreleripoll**
Refactor app to vue3
Made by creating a new vue3 project with create-vuetify and then moving all previous vue 2.7 (with composition API so nothing to change for composents themselves) to the new repo

Build time was up 20s when using dev server, now down to 400 ms

### CODE REVIEW

**Comment by pierreleripoll** [comment]
Also I removed old files not used anymore, such as hexagons CSV files or h3/d3 dependencies etc

File: frontend/.browserslistrc [hunk 1]
```diff
@@ -1,3 +1,4 @@
 > 1%
 last 2 versions
 not dead
+not ie 11
```
Comment (blueur) [inline frontend/.browserslistrc:4]: added by vue 3 by default ?

File: frontend/.eslintrc.js [hunk 2]
```diff
@@ -4,16 +4,11 @@ module.exports = {
     node: true,
   },
   extends: [
-    "plugin:vue/essential",
-    "eslint:recommended",
-    "@vue/typescript/recommended",
-    "plugin:prettier/recommended",
+    'plugin:vue/vue3-essential',
```
Comment (blueur) [inline frontend/.eslintrc.js:7]: better to use recommended https://eslint.vuejs.org/user-guide/#bundle-configurations

File: frontend/.eslintrc.js [hunk 3]
```diff
@@ -4,16 +4,11 @@ module.exports = {
...
+    'eslint:recommended',
+    '@vue/eslint-config-typescript',
   ],
-  parserOptions: {
-    ecmaVersion: 2020,
-  },
   rules: {
-    "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
-    "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+    'vue/multi-word-component-names': 'off',
```
Comment (blueur) [inline frontend/.eslintrc.js:12]: Don't want to keep that ?

**Review by blueur — APPROVED** [review]

File: frontend/.browserslistrc (hunk 1 above)
Comment (pierreleripoll) [reply frontend/.browserslistrc:4]: by create-vuetify 3 at least

File: frontend/.eslintrc.js (hunk 2 above)
Comment (pierreleripoll) [reply frontend/.eslintrc.js:7]: done in other branch

File: frontend/.eslintrc.js (hunk 3 above)
Comment (pierreleripoll) [reply frontend/.eslintrc.js:12]: yes good idea will add it back in other branch

File: frontend/.eslintrc.js (hunk 2 above)
Comment (pierreleripoll) [reply frontend/.eslintrc.js:7]: done in other branch

**Pull request merged**